### DISCUSSION TITLE
API Portal and cross project proxy endpoints

### DISCUSSION THREAD

**Discussion by giovanni-motterle-mia-care** [mia-platform/community#77] [Q&A] [answered]
Hi, I have created a cross project proxy (both projects belong to the same company in the same kubernetes cluster).
When I expose and endpoint for the proxy I can call it correctly form outside but I cannot see the documentation in the API portal.
In the endpoint setting is missing the flag "Show in API Portal", is this by design or a missing feature?

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, this is by design since there's no service in your project that is connected to that proxy, therefore the swagger aggregator couldn't know where the service replying that endpoint is in order to reach and fetch its oas. For this reason the "Show in API Portal" flag is not available.

**Reply by giovanni-motterle-mia-care** [reply]
actually in the endpoint configuration there is the namespace name and service name, is there some technical obstacle in gathering documentations that are served from another project?

**Reply by fredmaggiowski** [reply]
The swagger aggregator may not be able to reach the service (based on the network security policies) and also it does not know which is the documentation path for that service (since it is in another project). However this may be an interesting feature; i'd open this in the product backlog to be analysed in the future @albertotessarotto

**Reply by giovanni-motterle-mia-care** [reply]
ok, thanks!
